js-miniapp-sdk/src/types/responseTypes/AdResponse.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Response from native sdk for ads
+ * miniAppId is a GUID string
+ * adUnitId is string eg ca-app-pub-7941117952683310/1302674583
+ * startTime is ad's load start Time
+ * endTime is the ad's unload time
+ * provider is the active ad provider network selected
+ */
+export interface AdResponse {
+  miniAppId?: string;
+  adUnitId?: string;
+  startTime?: Date;
+  endTime?: Date;
```

Comment:
Why does the Mini App need to know `startTime` and `endTime` and how would this be retrieved on the native sides?

From what I can see, the native SDKs for Ad Mob don't have any interface for retrieving these values, so these would need to be calculated manually? In that case, the Mini App themselves would already know these values by storing a `Date` value before and after displaying the Ad, so it doesn't seem like they would need this.

Reply:
Removed!